papers/bradley_dice/55_bradley_dice.rst

```diff
@@ -0,0 +1,498 @@
+
+The generality of synced collections makes them broadly useful even outside the signac framework.
+The framework makes it easy for developers to create Pythonic APIs for data structures that might otherwise require significant additional implementation overhead.
+Crucially, synced collections support nesting as a core feature, something that could be quite difficult to handle for developers of custom collection types.
+Moreover, while the framework was originally conceived to support synchronization of an in-memory data structure with a resource on disk, it can just as easily be used to synchronize with another in-memory resource.
+One powerful example of this would be the use of a synced collection to provide a Pythonic API to a collection-like data structure implemented as a C or C++ extension module that could function like a Python dictionary with suitable plumbing but lacks the standard APIs expected of such a class.
+With the synced collections framework, creating a new class providing such an API is reduced to simply requiring the implementation of two straightforward methods defining the synchronization protocol.
+
+..
+    TODO: discuss independence from the rest of signac, possibility of releasing as a separate package?
+
+Related Software
+~~~~~~~~~~~~~~~~
+
+Are there other packages with related purposes? Vyas is not aware of any, the closest thing is Zict, a project Bradley pointed out a while ago.
+However, its scope is limited to composing mutable mappings.
+However, one natural question I'd expect from people is how this package to add a collection-like interface to some object compares to those objects directly implementing the interface.
+For example, I'd expect our closest comparison for a Redis-backed dict to be pyredis itself, which offers a dictionary-like API.
+I'd expect us to pretty much always be slower, but also to be much easier to work with and to support a lot more out-of-the-box (e.g.
+nested objects, buffering, and composition of data structures that may require more internal plumbing otherwise).
+
+
+Project Evolution
+-----------------
+
+The signac project has evolved from being an open-source project mostly developed and managed by the Glotzer Group at the University of Michigan, to being supported by over 30 contributors and committers/maintainers on 3 continents and with over 55 citations from academic and government research labs and 12 talks at large scientific, Python, and data science conferences.
+The growth in involvement with signac is the result of our focus on developing features based on user needs, as well as our efforts to transition signac users to signac contributors, through many initiatives in the past few years.
+Through encouraging users to become contributors, we ensure that signac addresses real users' needs.
+
+..
+    TODO: mention GSoC
```

Comment:
what more do we want to say about GSoC?

Reply:
I am not sure I want to say too much more. Most people will be familiar with it, and I am not sure that we will have unique things to say about it, though we had a great experience with it.